### PR TITLE
feat(gateway/oas-validation): OpenAPI 3.1.0 documentation

### PR DESCRIPTION
**Pull request by vm-001**
### Description

Update document for OAS-Validation plugin about OpenAPI 3.1.0

 

### Testing instructions

Preview link: https://deploy-preview-7310--kongdocs.netlify.app/hub/kong-inc/oas-validation/unreleased/

### Checklist 

- [x] Review label added 
- [x] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

https://github.com/Kong/kong-ee/pull/7833

### CODE REVIEW

**Comment by vm-001** [comment]
image

line break `</br>` is not working.

File: app/_hub/kong-inc/oas-validation/overview/_index.md [hunk 1]
```diff
@@ -2,9 +2,15 @@
...
-For JSON Schema draft 4 type schemas, see the [JSON Schema documentation](https://json-schema.org/) for details on the format and examples.
+Validate HTTP requests and responses against an OpenAPI Specification.
+
+The plugin supports both Swagger(v2) and OpenAPI(3.0.x) specifications, with the support of JSON Schema [Draft-04](https://json-schema.org/specification-links#draft-4). 
+
+{% if_plugin_version eq:3.7.x %}
+
+[OpenAPI 3.1.0](https://www.openapis.org/blog/2021/02/18/openapi-specification-3-1-released) is supported in Kong Gateway 3.7.0.0 with a new JSON Schema validator that supports Draft 2019-09.
+
+{% endif_plugin_version %}
```
Comment (lena-larionova) [inline app/_hub/kong-inc/oas-validation/overview/_index.md:13]:
```suggestion

{% if_version lte:3.6.x %}

The plugin supports both Swagger (v2) and OpenAPI (3.0.x) specifications, with the support of JSON Schema [Draft-04](https://json-schema.org/specification-links#draft-4). 

{% endif_version %}
{% if_version gte:3.7.x %}

The plugin supports both Swagger (v2) and OpenAPI (3.0.x and 3.1.0) specifications with a JSON Schema validator that supports [Draft 2019-09](https://json-schema.org/specification-links#draft-2019-09-(formerly-known-as-draft-8)).

{% endif_version %}
```

The plugin hub now supports `if_version` like the rest of the doc site.

File: app/_hub/kong-inc/oas-validation/overview/_index.md [hunk 2]
```diff
@@ -23,6 +29,20 @@ To enable the plugin, use one of the following methods:
 
 {% endif_plugin_version %}
 
+{% if_plugin_version eq:3.7.x %}
```
Comment (lena-larionova) [inline app/_hub/kong-inc/oas-validation/overview/_index.md:32]:
```suggestion
{% if_version gte:3.7.x %}
```
`gte` gives us "greater than or equal to", so the table will appear in subsequent versions too.

File: app/_hub/kong-inc/oas-validation/overview/_index.md [hunk 3]
```diff
@@ -23,6 +29,20 @@ To enable the plugin, use one of the following methods:
...
+
+| Category                        | Supported                      | Not supported                                                            |
+|---------------------------------|--------------------------------|--------------------------------------------------------------------------|
+| Request Body                    | application/json               | application/xml</br>multipart/form-data</br>text/plain</br>text/xml</br> |
+| Response Body                   | application/json               | -                                                                        |
+| Request Parameters              | path, query, header, cookie    | -                                                                        |
+| Schema                          | allOf</br>oneOf</br>anyOf</br> | -                                                                        |
+| Parameter Serialization         | style, explode                 | -                                                                        |
+
+{% endif_plugin_version %}
```
Comment (lena-larionova) [inline app/_hub/kong-inc/oas-validation/overview/_index.md:44]:
```suggestion
{% endif_version %}
```

**Review by lena-larionova — APPROVED** [review]
table is fixed now, lgtm

**Pull request merged**